stdlib/3/curses/ascii.pyi

```diff
@@ -0,0 +1,62 @@
+from typing import List, Union, overload, TypeVar
+
+_Ch = TypeVar('_Ch', str, int)
+
+NUL: int
+SOH: int
+STX: int
+ETX: int
+EOT: int
+ENQ: int
+ACK: int
+BEL: int
+BS: int
+TAB: int
+HT: int
+LF: int
+NL: int
+VT: int
+FF: int
+CR: int
+SO: int
+SI: int
+DLE: int
+DC1: int
+DC2: int
+DC3: int
+DC4: int
+NAK: int
+SYN: int
+ETB: int
+CAN: int
+EM: int
+SUB: int
+ESC: int
+FS: int
+GS: int
+RS: int
+US: int
+SP: int
+DEL: int
+
+controlnames: List[int]
+
+def isalnum(c: str) -> bool: ...
```

Comment:
all these `is*` functions take `Union[str, int]` (as mentioned in the previous review)